Remove-A-Submodule.md

```diff
@@ -0,0 +1,22 @@
+Removing a submodule is useful when it is no longer required. The steps below outline the removal of a submodule.
+
+#### Remove Submodule
+
+1. Delete the section referring to the submodule from the ```.gitmodules``` file
```

Comment:
Fix all the inline linking. See [**`Code Formatting`**](https://github.com/freecodecamp/freecodecamp/wiki/code-formatting)